agentos/component.py

```diff
@@ -196,18 +204,18 @@ def from_registry_file(
     @classmethod
     def from_class(
         cls,
-        managed_cls: Type[T],
+        managed_obj: Type[T],
```

Comment:
Good observation. I renamed the parameter to `class_obj` since `managed_cls` might make it seem like that specific Class object might become managed, which is how it used to work. Now the Component creates a new Class object for every call to `run()` or `get_object()`.